docs/setup.md

```diff
@@ -80,12 +80,41 @@ You will need to configure some listeners to make the bridge functional.
 ```
 
 At a minimum, you should bind the `webhooks` resource to a port and address. You can have multiple resources on the same
-port, or one on each.
+port, or one on each. Each listener MUST listen on a unique port.
 
 You will also need to make this port accessible to the internet so services like GitHub can reach the bridge. It
 is recommended to factor hookshot into your load balancer configuration, but currrently this process is left as an
 excercise to the user.
 
+In terms of API endpoints:
+
+- The `webhooks` resource handles resources under `/`, so it should be on it's own listener.
+- The `metrics` resource handles resources under `/metrics`.
+- The `provisioning` resource handles resources under `/v1/...`.
+
+#### Appservice listener
+
+<section class="notice">
+Please note that the appservice HTTP listener is configured <strong>seperately</strong> from the rest of the bridge due to lack of support
+in the upstream library. See <a href="https://github.com/turt2live/matrix-bot-sdk/issues/191">this issue</a> for details.
+</section>
+
+
+```yaml
+bridge:
+  # Basic homeserver configuration
+  #
+  domain: example.com
+  url: http://localhost:8008
+  mediaUrl: http://example.com
```

Comment:
perhaps it would be reasonable to explain:
- domain: "name" of your homeserver, as it appears in your MXIDs
- url: where hs is reachable, preferably internally i.e. localhost => docker needs adjustment
- mediaUrl: homeserver, but via potential load balancer etc to reach the media worker(?)
- port & bindAddress need adjustment/forwarding when using docker